docker/requirements.txt

```diff
@@ -8,4 +8,5 @@ nanomsg
 openpyxl
 pyexcel-ezodf
 aiohttp
+loguru
```

Comment:
Why did you choose `loguru` as the logging framework (instead of for example structlog). Just curious.